var/spack/repos/builtin/packages/intel-mkl/package.py

```diff
@@ -96,19 +96,19 @@ def lapack_libs(self):
     @property
     def scalapack_libs(self):
         libnames = ['libmkl_scalapack']
-        if self.spec.satisfies('^openmpi'):
+        root = self.spec.root
```

Comment:
maybe add a comment above this line why we do it (instead of doing `self.spec`) so that if new contributor come across this code, they will understand the logic